public/authz/utils.js

```diff
@@ -3,3 +3,14 @@ import chrome from 'ui/chrome';
 export const isKibanaApp = () => (window.location.pathname || '').endsWith('/app/kibana');
 
 export const isFeatureEnabled = (featureName) => chrome.getInjected(`keycloak.features.${featureName}`);
+
+export const isDirectiveExist = (module, directive) => module._invokeQueue
+    .filter(item => 'directive' === item[1])
+    .map(item => item[2][0])
+    .includes(directive);
+
+export const decorate = (module, directive, decorator) => {
```

Comment:
- Consider naming it `decorateDirective` instead of plain `decorate`
- `isDirectiveExist` can be changed to `directiveExists` and I guess there's no reason to export it.